package.json

```diff
@@ -14,9 +14,13 @@
     "start": "gulp server",
     "docs": "gulp docs",
     "watch": "lerna run --parallel watch",
-    "deploy": "lerna run deploy"
+    "deploy": "lerna run deploy",
+    "snyk-protect": "snyk protect",
+    "prepare": "npm run snyk-protect"
+  },
+  "dependencies": {
```

Comment:
should it be not a devDependency?

Reply:
Since this is the root of a monorepo it doesn't impact much, but yes.